grunt/svg_inline.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+var inline = require( "web-resource-inliner" );
+var a = require( "async" );
+
+module.exports = function ( grunt )
+{
+    grunt.registerTask( "svg_inline", "Inline SVG use", function ()
+    {
+        var done = this.async();
+        var src = [
+            "modules/*/templates/*.html",
+            "!modules/_app/templates/index.html"
```

Comment:
the index file is moved from `source` to `build` in the copy task, including it here would still lead to it being overwritten when running a build